SerialPrograms/Source/PokemonHome/Programs/PokemonHome_BoxSorting.cpp

```diff
@@ -242,6 +253,11 @@ std::ostream& operator<<(std::ostream& os, const std::optional<Pokemon>& pokemon
         os << "ball_slug:" << pokemon->ball_slug << " ";
         os << "gender:" << gender_to_string(pokemon->gender) << " ";
         os << "ot_id:" << pokemon->ot_id << " ";
+        os << "level:" << pokemon->level << " ";
+        os << "language:" << pokemon->language << " ";

```

Comment:
Try to keep the same order as other locations>

    uint32_t level = 0;
    std::string origin = "";
    std::string language = "";
    std::string nature = "";
    std::string ability = "";